pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -148,6 +150,14 @@ type KeySelector struct {
 	Key string `json:"key"`
 }
 
+// Trust store is used to enrich AdditionalFormats structure with
+// arbitrary password feature.
+type Truststore struct {
+	KeySelector `json:",inline"`
+	// Arbitrary password for particular store (JKS or PKCS12)
```

Comment:
```suggestion
	// Password to the truststore
```

I also think we should create two structs, one for JKS and another one for PKCS12. This will allow us to configure a static API default for the password. It might look like this for JKS (equivalent for PKCS12).

````go
type JKS struct {
    KeySelector `json:",inline"`

    // Password to JKS trust store
    //+kubebuilder:default=changeit
    Password *string `json:"password,omitempty"`
}
````

Note the pointer used for the optional field.